ecommerce/extensions/checkout/utils.py

```diff
@@ -33,7 +34,17 @@ def get_credit_provider_details(credit_provider_id, site_configuration):
         return None
 
 
-def get_receipt_page_url(site_configuration, order_number=None, override_url=None, disable_back_button=False):
+def _use_microfrontend_receipt(request):
+    """
+    Return whether the current request should use the ecommerce MFE receipt.
+    """
+    return(
```

Comment:
Very minor, sorry...

```suggestion
    return (
```

Reply:
ah thank you! I'm surprised pylint doesn't say anything :)